CMakeLists.txt

```diff
@@ -465,16 +465,19 @@ CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/SipiConfig.h.in ${CMAKE_CURRENT_SOURC
 # Here we get the ICC profiles from the Adobe website and prepare them for including
 # We just use the AdobeRGB1998.cc and the USWebCoatedSWOP.icc profiles for now...
 #
+unset(ICC CACHE)
 find_file(
     ICC
-    ${CMAKE_CURRENT_SOURCE_DIR}/icc.zip
+    icc.zip
+    ${CMAKE_CURRENT_SOURCE_DIR}
     NO_DEFAULT_PATH
 )
-if(ICC-NOTFOUND)
```

Comment:
this is explained in this [link](http://stackoverflow.com/questions/29657195/how-to-test-if-cmake-found-a-library-with-find-library)